src/index.ts

```diff
@@ -280,11 +280,14 @@ class Cents {
   }
 
   /**
-   * always show 2 decimal places
+   * show 2 decimal places unless specified
```

Comment:
we should add a note that this adds a dollar sign which could be unexpected unless you inspect the function